docs/scripts/generateRSSFeed.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+import { Feed } from 'feed';
+import { getAllBlogPosts, BlogPost } from 'docs/lib/sourcing';
+
+function getFeedOptions(siteUrl: string) {
+  return {
+    title: 'MUI - Blog',
+    description:
+      'Follow the MUI blog to learn about new product features, latest advancements in UI development, and business initiatives.',
+    id: `${siteUrl}/blog`,
+    link: `${siteUrl}/blog`,
+    language: 'en',
+    image: `${siteUrl}/static/logo.svg`,
+    favicon: `${siteUrl}/favicon.ico`,
+    copyright: `Copyright © ${new Date().getFullYear()} Material UI SAS.`,
+    feedLinks: {
+      rss2: `${siteUrl}/static/blog/feed/rss.xml`,
```

Comment:
Can you add a comment on why `rss2` is used? This could be useful for other who don't have experience with feed (like me).

Reply:
i used `rss2` because of reader-support, you can read this article for more information
[RSS Alternatives: Atom and JSON](https://thenewleafjournal.com/an-introduction-to-rss-and-other-feed-formats/#bnp_i_2)

and this question-answer
[should-i-provide-rss-or-atom-feeds](https://wordpress.stackexchange.com/questions/2922/should-i-provide-rss-or-atom-feeds)